transformations/suspecting_paraphraser/README.md

```diff
@@ -0,0 +1,27 @@
+# Suspecting Paraphraser 🦎  + ⌨️ → 🐍
+This paraphraser transforms a yes/no question into a ![tag question](https://www.englishclub.com/grammar/tag-questions.htm), which helps to add more question specific informality to the dataset. 
+
+Example: "Did the American National Shipment company really break its own fleet?" -> "The American National Shipment company really broke its own fleet, didn't it?"
```

Comment:
Is there an easy way to detect which qn is a yes/no question , so that we can only apply this transformation in those cases? [Was wondering how we would apply this transformation at scale on a corpus of questions]

Reply:
I made a filter for this - #95

Reply:
Thanks very much for addressing this!! Sorry I missed replying on this earlier....